src/components/ClubHome/Daily/DailyControl.js

```diff
@@ -16,81 +16,102 @@ const DailyControl = ({
-        </div>
-        <div>
-          {schedule.map((el, i) => {
-            return Date.parse(el.startDate) <= Date.parse(date) &&
-              Date.parse(date) <= Date.parse(el.endDate) ? (
-              <>
-                <br />
-                <span style={{ color: `${el.colorCode}` }} key={el.no}>
-                  {el.title}
-                </span>
-                <HiPencil onClick={() => onClickModify()} />
-                <FaTrashAlt
-                  onClick={() => {
-                    axios(
-                      `http://3.36.72.145:8080/api/club/schedule/1/${el.no}`,
-                      {
-                        method: "DELETE",
-                        headers: {
-                          "Content-type": "application/json; charset=utf-8",
-                          "x-auth-token": token,
-                        },
-                      }
-                    ).then((res) => getInfo());
-                  }}
-                />
-                {el.important ? (
-                  <AiFillStar
-                    onClick={() => {
-                      axios(
-                        `http://3.36.72.145:8080/api/club/schedule/1/${el.no}`,
-                        {
-                          method: "PATCH",
-                          headers: {
-                            "Content-type": "application/json; charset=utf-8",
-                            "x-auth-token":
-                              "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6InRlc3QxIiwibmFtZSI6InRlc3QxIiwiY2x1Yk51bSI6IlsxXSJ9.1u6k5cJuaUlZj14CJJZiI8guHnlZXf1uuU6vZjl9jNk",
-                          },
-                          data: { important: 0 },
-                        }
-                      ).then((res) => getInfo());
-                    }}
-                  />
-                ) : (
-                  <AiOutlineStar
-                    onClick={() => {
-                      axios(
-                        `http://3.36.72.145:8080/api/club/schedule/1/${el.no}`,
-                        {
-                          method: "PATCH",
-                          headers: {
-                            "Content-type": "application/json; charset=utf-8",
-                            "x-auth-token":
-                              "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6InRlc3QxIiwibmFtZSI6InRlc3QxIiwiY2x1Yk51bSI6IlsxXSJ9.1u6k5cJuaUlZj14CJJZiI8guHnlZXf1uuU6vZjl9jNk",
-                          },
-                          data: { important: 1 },
-                        }
-                      ).then((res) => getInfo());
-                    }}
-                  />
-                )}
-              </>
-            ) : null;
-          })}
+      <div
+        className={styles.wrap}
+        onClick={() => setPop(0)}
+        //회색 클릭시 꺼지는 기능 구현해야댐
+      >
+        <div className={styles.modal} onClick={(e) => e.stopPropagation()}>
+          <div className={styles.header}>
+            <h3>{date} 일정</h3>
+          </div>
```

Comment:
단순한 컴포넌트이기는 하지만 이런것도 DailyControlHeader.js라는 컴포넌트로 따로 빼서 넣어주면 
헤더 바디 푸터 이런 요소들을 한 눈에 알아보기 쉬울거같아요
```suggestion
          <DailyControlHeader />
```